File: hackerrank/java/30-days-of-code/day-4-class-vs-instance.java
import java.io.*;
import java.util.*;

public class Person {
    private int age;

    public Person(int initialAge) {
        this.age = 0;
        if (initialAge < 0) {
            System.out.println("Age is not valid, setting age to 0.");
        } else
            this.age = initialAge;
    }

    public void amIOld() {
        if (this.age < 13)
            System.out.println("You are young.");
        else if (this.age >= 13 && this.age < 18)
            System.out.println("You are a teenager.");
        else
            System.out.println("You are old.");
    }

    public void yearPasses() {
        this.age++;
    }
}

File: hackerrank/python/30-days-of-code/day-4-class-vs-instance.py
class Person:
    age = 0

    def __init__(self, initialAge):
        if initialAge >= 0:
            self.age = initialAge
        else:
            print('Age is not valid, setting age to 0.')
            self.age = 0

    def amIOld(self):
        if self.age < 13:
            print('You are young.')
        elif self.age < 18:
            print('You are a teenager.')
        else:
            print('You are old.')

    def yearPasses(self):
        self.age = self.age + 1
